gcloud-java-resourcemanager/src/test/java/com/google/gcloud/resourcemanager/ProjectTest.java

```diff
@@ -0,0 +1,214 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.resourcemanager;
+
+import static org.easymock.EasyMock.createStrictMock;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.expectLastCall;
+import static org.easymock.EasyMock.replay;
+import static org.easymock.EasyMock.verify;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+import com.google.gcloud.resourcemanager.Policy.Binding;
+import com.google.gcloud.resourcemanager.Policy.Member;
+import com.google.gcloud.resourcemanager.Policy.RoleType;
+import com.google.gcloud.spi.ResourceManagerRpc.Permission;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ProjectTest {
+  private static final String ID = "project-id";
+  private static final String NAME = "myProj";
+  private static final Map<String, String> LABELS = new HashMap<String, String>();
+  static {
+    LABELS.put("k1", "v1");
+    LABELS.put("k2", "k2");
+  }
+  private static final Long NUMBER = 123L;
+  private static final Long CREATE_TIME_MILLIS = 123456789L;
+  private static final ProjectInfo.State STATE = ProjectInfo.State.DELETE_REQUESTED;
+  private static final ResourceId PARENT = ResourceId.of("owner-id", ResourceId.Type.ORGANIZATION);
+  private static final ProjectInfo PROJECT_INFO =
+      ProjectInfo.builder(ID)
+          .name(NAME)
+          .labels(LABELS)
+          .number(NUMBER)
+          .createTimeMillis(CREATE_TIME_MILLIS)
+          .state(STATE)
+          .parent(PARENT)
+          .build();
+  private static final Binding OWNER_BINDING;
+  private static final Binding EDITOR_BINDING;
+  private static final Binding VIEWER_BINDING;
+  static {
+    List<Member> ownerMemberList = new ArrayList<>();
+    List<Member> editorMemberList = new ArrayList<>();
+    List<Member> viewerMemberList = new ArrayList<>();
+
+    ownerMemberList.add(Member.user("first-owner@email.com"));
+    ownerMemberList.add(Member.group("group-of-owners@email.com"));
+    OWNER_BINDING = Policy.Binding.builder().role(RoleType.OWNER).members(ownerMemberList).build();
+    editorMemberList.add(Member.serviceAccount("editor@someemail.com"));
+    EDITOR_BINDING =
+        Policy.Binding.builder().role(RoleType.EDITOR).members(editorMemberList).build();
+    viewerMemberList.add(Member.serviceAccount("app@someemail.com"));
+    viewerMemberList.add(Member.user("viewer@email.com"));
+    VIEWER_BINDING =
+        Policy.Binding.builder().role(RoleType.VIEWER).members(viewerMemberList).build();
+  }
+  private static final Policy POLICY =
+      Policy.builder()
```

Comment:
It's fine as is. I based my comment on [this](https://github.com/GoogleCloudPlatform/gcloud-java/blob/master/gcloud-java-storage/src/main/java/com/google/gcloud/spi/DefaultStorageRpc.java#L149).